### ISSUE TITLE
🕵️ Searching with empty input leads to infinite loading

### ISSUE THREAD

**Issue by lukeomalley** [canikwe/swa_frontend#7]
Fix this asap its ruining my UX

**Comment by lukeomalley** [comment]
Any updates?? I really need this fixed.

**Comment by canikwe** [comment]
> Any updates?? I really need this fixed.

@lukeomalley Hey, thanks for reaching out! Yeah, will look into this asap. In the meantime, can you outline the specific steps you took before you encountered the bug? Any screenshots you have would also be really helpful. Cheers!

**Comment by lukeomalley** [comment]
After cloning and starting the client and server I realized that I do not have an open weather api key! 😢

![Screen Shot 2022-02-11 at 10 29 08 AM](https://user-images.githubusercontent.com/45078832/153620443-af0d09a9-8136-40cc-8444-3e797276cb00.png)
![Screen Shot 2022-02-11 at 10 28 07 AM](https://user-images.githubusercontent.com/45078832/153620446-55e4a534-368d-443d-ae9d-f4ee5fea78e0.png)
�

**Comment by canikwe** [comment]
@lukeomalley Yeah you're going to need that 🔑  to test locally. I was able to look into this on my end. Seems like user error? 🤷🏽  Would you mind testing (not locally) again?

**Comment by lukeomalley** [comment]
@canikwe Hey I'm still stuck on this. Any updates on getting me that key?
